tap_kafka/__init__.py

```diff
@@ -77,6 +78,7 @@ def generate_config(args_config):
 
         # Add optional parameters with defaults
         'primary_keys': args_config.get('primary_keys', {}),
+        'initial_start_time': args_config.get('initial_start_time', DEFAULT_INITIAL_START_TIME),
```

Comment:
Can we add validation for this `initial_start_time` in discovery mode to check that it's a correct ISO formatted date?

Reply:
I added `validate_config` function that runs before the discovery or normal the sync starts.

Reply:
Cool, thanks